ExotischNederland/Models/Answer.cs

```diff
@@ -1,13 +1,43 @@
-﻿using System;
+﻿using ExotischNederland.DAL;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace ExotischNederland.Models
 {
-    internal class Answer
+    public class Answer
     {
-        private string tablename = "Answer";
+        public int Id { get; set; }
+        public Question Question { get; set; }
+        public string Text { get; set; }
+        public bool Correct { get; set; }
+
+        private Answer(int id, Question question, string text, bool correct)
```

Comment:
Please use the constructor used in other files (with the Dictionary<string, object>